docs/reference/index-modules/index-sorting.asciidoc

```diff
@@ -0,0 +1,105 @@
+[[index-modules-index-sorting]]
+== Index Sorting
+
+When creating a new index in elasticsearch it is possible to configure how the Segments
+inside each Shard will be sorted. By default Lucene does not apply any sort and uses the
+internal _doc_id_ to do the ordering.
+The `index.sort.*` settings define which fields should be used to sort the documents inside each Segment.
+
+[WARNING]
+nested fields are not compatible with index sorting because they rely on the default doc_id sorting.
```

Comment:
s/nested/Nested/ and maybe `s/on the default doc_id sorting/on the assumption that nested documents are stored in contiguous doc ids, which can be broken by index sorting/`?